### ISSUE TITLE
controller is not launched with CRIO

### ISSUE THREAD

**Issue by fabiand** [kubevirt/kubevirt#872]
Launching an image using crio fails:

```
Warning  Failed                 7s (x6 over 1m)  kubelet, minikube  \
Error: container has runAsNonRoot and image will run as root
```

reproducable on

```
minikube start --container-runtime=cri-o --network-plugin=cni \
  --bootstrapper=kubeadm --vm-driver=kvm2
```

**Comment by fabiand** [comment]
Hm.
The container spec requires a non-root user, and the image has a non-root user set. But CRIO still complains. I wonder why.

**Comment by fabiand** [comment]
`runc list` in minikube reveals that all containers are run as root.

**Comment by mrunalp** [comment]
@fabiand What version of cri-o do you see running? We have fixed the issue of all running is root in recent versions of 1.9 in cri-o. You should run 1.9.11 and let us know if you see the same issue.

**Comment by fabiand** [comment]
Yes, it was experienced on crio 1.6.

I'm waiting that minikube is seeing an update.

**Comment by fabiand** [comment]
This is fixed with minikube v0.26.0 which has

```
# crictl -r /var/run/crio.sock  version
Version:  0.1.0
RuntimeName:  cri-o
RuntimeVersion:  1.10.0
RuntimeApiVersion:  v1alpha1
```
